src/actinia_core/testsuite.py

```diff
@@ -451,7 +480,8 @@ def create_vector_layer(self, location, mapset, vector, region, parameter):
         )
         self.waitAsyncStatusAssertHTTP(
             rv,
-            headers=self.admin_auth_header,
+            headers=self.user_auth_header,
+            # headers=self.admin_auth_header,
```

Comment:
Could this lead to errors?

Reply:
It has not lead to errors by other tests